repos/spack_repo/builtin/packages/affinity/package.py

```diff
@@ -21,8 +22,19 @@ class Affinity(CMakePackage, CudaPackage):
 
     variant("mpi", default=False, description="Build MPI support")
     variant("rocm", default=False, description="Build ROCm Support")
+    variant("cuda", default=False, description="Build CUDA Support")
```

Comment:
The variants `rocm` and `cuda` are inherited from the base classes. Since the default values are the same, we can remove them from the definition here.